src/test/java/com/savvato/tribeapp/services/UserRoleMapServiceImplTest.java

```diff
@@ -133,4 +133,25 @@ public void testRemoveIncorrectRolesFromUser() {
 
         assertThat(returnedFalse).isFalse();
     }
+
+    @Test
+    public void testRemoveOneRoleAccountHolder() {
+        // create a new array list with ADMIN, ACCOUNT_HOLDER, PHRASE_REVIEWER in it
```

Comment:
Update this comment to reflect the code.

that's one thing to remember, is that comments can become stale: the code they reference can change without the comment itself changing. So, two things.. one, I try to make the code itself through clear names, etc, describe what is happening. Two, always be sure the comments that are present are indeed helpful.